tsconfig.json

```diff
@@ -11,21 +24,9 @@
     "noPropertyAccessFromIndexSignature": true,
     "noUncheckedIndexedAccess": true,
     "noUnusedLocals": true,
-    "noUnusedParameters": true,
-
-    // Modules
-    "allowArbitraryExtensions": true,
-    "module": "ES2022",
-    "moduleResolution": "node",
-
-    // Emit
-    "preserveConstEnums": true,
-    "sourceMap": true,
-
-    // Interop Constraints
-    "allowSyntheticDefaultImports": true
+    "noUnusedParameters": true
   },
```

Comment:
I tried to revert to my previous configuration, everythings still work, can we change it back? If we want to update the configuration, better do it in separate PR.

Reply:
You're right, because I copied the wrong one